src/components/Search/index.tsx

```diff
@@ -134,19 +137,6 @@ function Search({queryJSON, policyIDs, isCustomQuery}: SearchProps) {
         setDeleteExpensesConfirmModalVisible(true);
     };
 
-    useEffect(() => {
```

Comment:
this was a duplicate of useEffect in line 96 - probably someone made a mistake when refactoring